### ISSUE TITLE
Integration vs channel

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#2044]
It was pointed out that some variable names refer to "integration", but are actually referring to "channel" values.

Follow up to https://github.com/prebid/prebid-server/issues/1428

In config/accounts.go
```
// IntegrationType enumerates the values of integrations Prebid Server can configure for an account
type IntegrationType string

// Possible values of integration types Prebid Server can configure for an account
const (
	IntegrationTypeAMP   IntegrationType = "amp" // should be "ChannelAMP"
	IntegrationTypeApp   IntegrationType = "app"  // should be "ChannelApp"
	IntegrationTypeVideo IntegrationType = "video"   // should be "ChannelVideo"
	IntegrationTypeWeb   IntegrationType = "web"  // should be "ChannelWeb"
)
```
Similarly, in PBS-Java config, privacy.*.integration-enabled should be "channel-enabled"
```
          integration-enabled:     // "channel-enabled" please
            video: true
            web: true
            app: true
            amp: true
```

This change would be one that host companies would need to be aware of. Will discuss in the next PBS meeting.

**Comment by bretg** [comment]
The first phase of this released with PBS-Java 1.79. PBS-Java users should consider checking their databases and migrating "integration-enabled" to "channel-enabled". At some point in the near future, this update will be required.

**Comment by bsardo** [comment]
`IntegrationType` has been changed to `ChannelType` in PBS-Go and the corresponding instances have been renamed:
```
// ChannelType enumerates the values of integrations Prebid Server can configure for an account
type ChannelType string

// Possible values of channel types Prebid Server can configure for an account
const (
	ChannelAMP   ChannelType = "amp"
	ChannelApp   ChannelType = "app"
	ChannelVideo ChannelType = "video"
	ChannelWeb   ChannelType = "web"
	ChannelDOOH  ChannelType = "dooh"
)
```

Both `integration-enabled` and `channel-enabled` have been supported for about a year at this point and we recently removed support for `integration-enabled` as part of the 2.0 release initiative.
